### DISCUSSION TITLE
Possibility to run host by host instead of parallel

### DISCUSSION THREAD

**Discussion by fliespl** [deployphp/deployer#3891] [Features]
I am curious if it's possible to execute specific task on each host one by one instead of parallel.

Use case is i.e. glusterfs which end up with stale file handle

[xxx.htz.srv.xxx.pl] err grep: /var/www/vhosts/xxx/shared/.env.local: Stale file handle

[once() option doesn't always fill our needs]

**Comment by antonmedv** [comment]
Deployer runs a task in parallel on each host. To limit it to run only on one host at a time use `--limit`

    dep deploy --limit 1

**Reply by fliespl** [reply]
Oh thanks. So it will work like that:

host1 - deploy:info - start
host1 - deploy:info - end
host2 - deploy:info - start
host2 - deploy:info - end
[...]
host1 - deploy:success - start
host1 - deploy:success - end
host2 - deploy:success - start
host2 - deploy:success - end

or 


host1 - deploy:info - start
host1 - deploy:info - end
[...]
host1 - deploy:success - start
host1 - deploy:success - end

host2 - deploy:info - start
host2 - deploy:info - end
[...]
host2 - deploy:success - start
host2 - deploy:success - end

**Reply by antonmedv** [reply]
One task per host.
